### PR TITLE
Add more AWS regions

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Fixes iterative/cml#437

### CODE REVIEW

**Comment by 0x2b3bfa0** [comment]
@DavidGOrtega, please take a look before I start cloning the images.

**Review by DavidGOrtega — APPROVED** [review]
Lgtm 🚀

**Comment by DavidGOrtega** [comment]
@0x2b3bfa0 probably we will have to open some regions that would be closed. We might ask @JIoJIaJIu if it does not succeed.

**Comment by 0x2b3bfa0** [comment]
```console
AuthFailure: AWS was not able to validate the provided access credentials
	status code: 401, request id: a7e1····-····-····-····-········c1c8
```

**Comment by DavidGOrtega** [comment]
@0x2b3bfa0 whats the region that fails?

**Comment by 0x2b3bfa0** [comment]
🤦  Oops! Sorry, I missed the important detail: `af-south-1`

**Comment by DavidGOrtega** [comment]
@0x2b3bfa0 have you asked @JIoJIaJIu ?

File: iterative/aws/provider.go
Comment (0x2b3bfa0) [inline iterative/aws/provider.go:360]:
```suggestion
	"us-east-2",
	"us-east-1",
	"us-west-1",
	"us-west-2",
	"ap-south-1",
	"ap-northeast-3",
	"ap-northeast-2",
	"ap-southeast-1",
	"ap-southeast-2",
	"ap-northeast-1",
	"ca-central-1",
	"eu-central-1",
	"eu-west-1",
	"eu-west-2",
	"eu-west-3",
	"eu-north-1",
	"sa-east-1",
```

**Conflicting regions**

* af-south-1	
* ap-east-1
* eu-south-1
* me-south-1

@JIoJIaJIu, may we opt-in to these regions for deploying the CML machine image?

Comment (DavidGOrtega) [reply iterative/aws/provider.go:360]: @0x2b3bfa0 can we merge this or we still have those regions with issues?

Comment (0x2b3bfa0) [reply iterative/aws/provider.go:360]: @DavidGOrtega, please let me try again, I don't know if the missing regions have been enabled.

Comment (0x2b3bfa0) [reply iterative/aws/provider.go:360]: @DavidGOrtega & @JIoJIaJIu, it looks like they haven't been opted-in yet. 🤔

**Comment by 0x2b3bfa0** [comment]
@DavidGOrtega, this pull request can be merged as-is for now, but we're still missing the following opt-in regions:

- af-south-1
- ap-east-1
- eu-south-1
- me-south-1 

I've removed them for now, but we can also add them to the list as soon as @JIoJIaJIu enables them. It looks that this is the only missing step.

**Comment by 0x2b3bfa0** [comment]
@DavidGOrtega, shall we merge this pull request so it doesn't go stale?

**Comment by DavidGOrtega** [comment]
@0x2b3bfa0 yes, here we did not have a clear manager since we do not have releasing process right now and we have been merging ourselves once the approve is done against master, I mean, I thought that you were doing it.

But lets do a proper release like CML

**Comment by 0x2b3bfa0** [comment]
@DavidGOrtega, I asked you because you approved it prior to the opt-in region issue, but we can adopt whatever merging strategy you choose. I tend to rely on you for merging my pull requests as an additional sanity check. 😄

**Comment by DavidGOrtega** [comment]
Ok got it. Yep the PR is still approved after the change. We shoul open a ticket to not forget the other regions.

**Comment by 0x2b3bfa0** [comment]
@DavidGOrtega, for the record: https://github.com/iterative/terraform-provider-iterative/issues/100

**Pull request merged**
